wlauto/external/uiauto/project.properties

```diff
@@ -11,4 +11,4 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-17
+target=android-18
```

Comment:
Regarding the .properities

   - I don't see a commit with that change; have you pushed it?
   - Are *certain* that no APIs used by existing automations remove/renamed/changed between levels 17 and 18? If not please rebuild everything to make sure.